digital_image_processing/index_calculation.py

```diff
@@ -1,651 +1,525 @@
+                Wavelength Range 635 nm to 700 nm
+            * blue
+                https://en.wikipedia.org/wiki/Color
+                Wavelength Range 450 nm to 490 nm
+            * green
+                https://en.wikipedia.org/wiki/Color
+                Wavelength Range 520 nm to 560 nm
+
+                
+        # Implemented index list
+                #"abbreviationOfIndexName" -- list of channels used
+
+                #"ARVI2"            --  red, nir
+                #"CCCI"             --  red, redEdge, nir
+                #"CVI"              --  red, green, nir
+                #"GLI"              --  red, green, blue
+                #"NDVI"             --  red, nir
+                #"BNDVI"            --  blue, nir
+                #"redEdgeNDVI"      --  red, redEdge
+                #"GNDVI"            --  green, nir
+                #"GBNDVI"           --  green, blue, nir
+                #"GRNDVI"           --  red, green, nir
+                #"RBNDVI"           --  red, blue, nir
+                #"PNDVI"            --  red, green, blue, nir
+                #"ATSAVI"           --  red, nir
+                #"BWDRVI"           --  blue, nir
+                #"CIgreen"          --  green, nir
+                #"CIrededge"        --  redEdge, nir
+                #"CI"               --  red, blue
+                #"CTVI"             --  red, nir
+                #"GDVI"             --  green, nir
+                #"EVI"              --  red, blue, nir
+                #"GEMI"             --  red, nir
+                #"GOSAVI"           --  green, nir
+                #"GSAVI"            --  green, nir
+                #"Hue"              --  red, green, blue
+                #"IVI"              --  red, nir
+                #"IPVI"             --  red, nir
+                #"I"                --  red, green, blue
+                #"RVI"              --  red, nir
+                #"MRVI"             --  red, nir
+                #"MSAVI"            --  red, nir
+                #"NormG"            --  red, green, nir
+                #"NormNIR"          --  red, green, nir
+                #"NormR"            --  red, green, nir
+                #"NGRDI"            --  red, green
+                #"RI"               --  red, green
+                #"S"                --  red, green, blue
+                #"IF"               --  red, green, blue
+                #"DVI"              --  red, nir
+                #"TVI"              --  red, nir
+                #"NDRE"               --  redEdge, nir
+
+        #list of all index implemented
+            #allIndex = ["ARVI2", "CCCI", "CVI", "GLI", "NDVI", "BNDVI", "redEdgeNDVI", "GNDVI", "GBNDVI", "GRNDVI", "RBNDVI", "PNDVI", "ATSAVI", "BWDRVI", "CIgreen", "CIrededge", "CI", "CTVI", "GDVI", "EVI", "GEMI", "GOSAVI", "GSAVI", "Hue", "IVI", "IPVI", "I", "RVI", "MRVI", "MSAVI", "NormG", "NormNIR", "NormR", "NGRDI", "RI", "S", "IF", "DVI", "TVI", "NDRE"]
```

Comment:
These lines should be wrapped so that they can be read on GitHub with left-right scrolling.